src/sqlfluff/dialects/dialect_spark3.py

```diff
@@ -794,6 +794,53 @@ class UseDatabaseStatementSegment(BaseSegment):
     )
 
 
+# Data Manipulation Statements
+@spark3_dialect.segment()
+class InsertTableStatement(BaseSegment):
```

Comment:
Should this not just update `InsertStatementSegment`?

```suggestion
@spark3_dialect.segment(replace=True)
class InsertStatementSegment(BaseSegment):
```

Reply:
Yes, this makes sense. I'll make the update